src/datanode/src/datanode.rs

```diff
@@ -240,6 +240,20 @@ impl Datanode {
     pub fn get_instance(&self) -> InstanceRef {
         self.instance.clone()
     }
+
+    pub async fn shutdown_instance(&self) -> Result<()> {
```

Comment:
Not necessary to be public.
```suggestion
   async fn shutdown_instance(&self) -> Result<()> {
```